bundles/org.openhab.binding.goveelan/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="goveelan"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="govee-light">
+		<label>Govee light thing</label>
+		<description>Govee light controllable via LAN API</description>
+
+		<channels>
+			<channel id="switch" typeId="system.power"/>
```

Comment:
This channel is not needed. `OnOffType` commands should be supported by the `brightness` channel, and a `Switch` item can then be linked to the `brightness` channel to achieve the same.

Reply:
I removed the system.power channel but I have a problem here with updating the brightness channel regarding on/off: I am able to detect the input "ON/OFF" from the channel brightness and send the particular command and also updating the brightness via a brightness item works.

<img width="635" alt="image" src="https://github.com/openhab/openhab-addons/assets/5937600/1e331d18-f400-4cf9-92bf-5d0fe1b54676">

However, when I get the status response from the device and update it like so

<img width="909" alt="image" src="https://github.com/openhab/openhab-addons/assets/5937600/510aefa5-9b4e-4be8-947f-c85dde699965">

the OnOfType simply overwrites the brightness as well and is either 0 or 100, hence the brightness value is wrong now. Shouldn't updateState() with an OnOffType ignore the PercentType information internally?